app/src/layouts/common/sidebar/organizationsControl/organizationsPopover/organizationsPopover.scss

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.organizations-popover {
+  width: 216px;
+  background-color: $COLOR--darkmode-gray-400;
+}
+
+.all-organizations {
+  border-bottom: 1px solid $COLOR--almost-black;
+  padding: 8px 0;
+  margin-bottom: 8px;
+  margin-right: 8px;
+}
+
+.all-organizations-link {
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  color: $COLOR--darkmode-gray-100;
+  text-decoration: none;
+  text-align: left;
+  font-size: 13px;
+  line-height: 20px;
+  font-family: $FONT-ROBOTO-MEDIUM;
+}
+
+.scroll-wrapper {
+  > div:last-child {
+    width: 4px !important;
```

Comment:
Same here about `!important` directive. Pls check all cases from this PR.

Reply:
The scroll component is used by the importer, there is no other option.